task1/Bicycle.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task1
+{
+    class Bicycle : Vehicle
+    {
+        public BicycleType Type { get; set; }
```

Comment:
I believe that type of a bicycle can't be changed. So lets leave getter only.
It belongs to BrakeType too.